manager/controllers/advisory_detail.go

```diff
@@ -240,21 +214,21 @@ func PreloadAdvisoryCacheItems() {
 	progress, count := utils.LogProgress("Advisory detail cache preload", logProgressDuration, int64(len(advisoryNames)))
 
 	for _, advisoryName := range advisoryNames {
-		_, err = getAdvisoryV2(database.Db, advisoryName, true)
+		_, err = getAdvisory(database.Db, advisoryName, true)
 		if err != nil {
-			utils.LogError("advisoryName", advisoryName, "err", err.Error(), "can not re-load item to cache - V2")
+			utils.LogError("advisoryName", advisoryName, "err", err.Error(), "can not re-load item to cache - V3")
```

Comment:
we don't need this information that it is for V3, `can not re-load item to cache` would be enough but feel free to leave it as it is since it is only log for ourselves